### ISSUE TITLE
Unit Option Does Not Work

### ISSUE THREAD

**Issue by ayberkcal** [eopeter/flutter_mapbox_navigation#304]
Problem: I want to use _Kilometer_ metric instead of _Mile_ on screen and voice directions. However it displayed _mil_ and voice directions with _mil_

Plugin version: ^0.1.9

Mapbox option like that:
```
MapBoxOptions navigationOption = MapBoxNavigation.instance.getDefaultOptions()
    ..simulateRoute = true
    ..allowsUTurnAtWayPoints = true
    ..mode = MapBoxNavigationMode.drivingWithTraffic
    ..units = VoiceUnits.metric
    ..language = "en"
    ..bannerInstructionsEnabled = false;
```

**Comment by eopeter** [comment]
try the latest version 0.2.0 to see if this issue persist

**Comment by ortegaxc** [comment]
Hi @eopeter. i have the same issue with version 0.2.0.

![Captura de pantalla 2023-09-26 a la(s) 14 47 45](https://github.com/eopeter/flutter_mapbox_navigation/assets/65456053/bac66a16-4ca9-4446-a448-977fda9f1dc1)

![Captura de pantalla 2023-09-26 a la(s) 14 48 27](https://github.com/eopeter/flutter_mapbox_navigation/assets/65456053/a045f597-f52f-4b5c-b60c-4d467f2b091c)

![Captura de pantalla 2023-09-26 a la(s) 14 49 49](https://github.com/eopeter/flutter_mapbox_navigation/assets/65456053/33a95395-30ae-46b0-b28a-6394c83e271d)

**Comment by eopeter** [comment]
will look into this soon

**Comment by eopeter** [comment]
resolved in latest version 0.2.2
